### PR TITLE
Feature/contact upload widget

### PR DESCRIPTION
**Pull request by Tim-M-S**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested by the author
- [x] Changes have been tested by the reviewer

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/js/src/app/store/modules/ContactFormModule.js [hunk 1]
```diff
@@ -201,6 +212,37 @@ const actions =
...
+
+        for (const fileInput of fileInputs)
+        {
+            for (const file of fileInput.files)
+            {
+                formData.append("fileData[]", file);
+            }
+        }
+
+        formData.append("rcaptchaToken", recaptchaToken);
```
Comment (stentrop) [inline resources/js/src/app/store/modules/ContactFormModule.js]:
```suggestion
        formData.append("recaptchaToken", recaptchaToken);
```

**Review by stentrop — CHANGES_REQUESTED** [review]

File: resources/js/src/app/store/modules/ContactFormModule.js [hunk 2]
```diff
@@ -141,39 +141,50 @@ const actions =
...
+                                        disableForm(event.target, false);
+                                        response.error.message = response.error.message || TranslationService.translate("Ceres::Template.contactSendFail");
+                                        NotificationService.error(response.error);
+                                    });
+                            },
+                            (error) =>
+                            {
+                                resetRecaptcha(recaptchaEl);
+                                disableForm(event.target, false);
+                                response.error.message = response.error.message || TranslationService.translate("Ceres::Template.contactFileUploadFail");
```
Comment (stentrop) [inline resources/js/src/app/store/modules/ContactFormModule.js:185]: Hier gibt es kein response

**Review by stentrop — DISMISSED** [review]

File: resources/js/src/app/store/modules/ContactFormModule.js [hunk 3]
```diff
@@ -141,39 +141,50 @@ const actions =
...
+                                    })
+                                    .fail(response =>
+                                    {
+                                        resetRecaptcha(recaptchaEl);
+                                        disableForm(event.target, false);
+                                        response.error.message = response.error.message || TranslationService.translate("Ceres::Template.contactSendFail");
+                                        NotificationService.error(response.error);
+                                    });
+                            },
+                            (error) =>
```
Comment (lks-mtzn) [inline resources/js/src/app/store/modules/ContactFormModule.js]:
```suggestion
                            (response) =>
```

Comment (Tim-M-S) [reply resources/js/src/app/store/modules/ContactFormModule.js]: done

File: resources/js/src/app/store/modules/ContactFormModule.js (hunk 1 above)
Comment (Tim-M-S) [reply resources/js/src/app/store/modules/ContactFormModule.js]: done

File: resources/js/src/app/store/modules/ContactFormModule.js (hunk 2 above)
Comment (Tim-M-S) [reply resources/js/src/app/store/modules/ContactFormModule.js:185]: done

**Review by fmutschler — DISMISSED** [review]

**Review by cjohannsen — APPROVED** [review]

**Pull request merged**